service.py

```diff
@@ -60,6 +70,10 @@ def showErrorNotification(message):
 def getParams():
     result = {}
```

Comment:
If you are already on it, can improve the current code a bit and rename this local variable from `result` to `params` to avoid confusion with the other global result variable we have...